CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 - Minor: Don't show update button for nightly builds on macOS and Linux, this was already the case for Windows (#2163, #2164)
 - Minor: Tab and split titles now use display/localized channel names (#2189)
 - Minor: Add a setting to limit the amount of historical messages loaded from the Recent Messages API (#2250, #2252)
+- Minor: Color mentions to match the mentioned users (#1963, #2121, #2284)
```

Comment:
I would classify this as a major change, please also include information that this feature is enabled by default and can be disabled in the Settings window through the "Color @usernames" setting